cpp/lib/include/Zeder.h

```diff
@@ -405,7 +405,7 @@ class EndpointDownloader {
 public:
     virtual ~EndpointDownloader() = default;
 public:
-    virtual bool download(EntryCollection * const collection) = 0;
+    virtual bool download(EntryCollection * const collection, bool disable_cache_mechanism) = 0;
```

Comment:
This should be a const bool. Also it should have false as a default value, so we only need to override it if we explicitly want to turn the Cache off.